lib/buildFactory.js

```diff
@@ -268,10 +269,15 @@ class BuildFactory extends BaseFactory {
 
                     const build = await super.create(modelConfig);
 
-                    await steps.reduce(
-                        (p, step) => p.then(() => stepFactory.create({ buildId: build.id, ...step })),
-                        Promise.resolve()
-                    );
+                    // eslint-disable-next-line no-restricted-syntax
+                    for (const step of steps) {
+                        try {
+                            // eslint-disable-next-line no-await-in-loop
+                            await stepFactory.create({ buildId: build.id, ...step });
+                        } catch (err) {
+                            logger.error(`Error in steps -BuildId:${build.id}-step:${step}`, err);
```

Comment:
Could have error message something like below to be more consistent with https://github.com/screwdriver-cd/screwdriver/blob/master/plugins/builds/index.js#L1346-L1347, but not a big deal if we leave it as-is

```suggestion
                            logger.error(`Error in buildFactory create steps - buildId:${build.id}-step:${step}`, err);
```

Reply:
Thank you review.
I modify error message.